### PR TITLE
Created mediator for Club

### PR DESCRIPTION
**Pull request by andrii52**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: EPlast/EPlast.BLL/Commands/Club/ArchiveCommand.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using MediatR;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace EPlast.BLL.Commands.Club
+{
+    public class ArchiveCommand : IRequest
+    {
+        public int CLubId { get; set; }
```
Comment (ostappppppp) [inline EPlast/EPlast.BLL/Commands/Club/ArchiveCommand.cs:10]: Typo here
`ClubId`

File: EPlast/EPlast.BLL/Commands/Club/UnArchiveCommand.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EPlast.BLL.Commands.Club
+{
+    public class UnArchiveCommand : IRequest
+    {
+        public int CLubId { get; set; }
```
Comment (ostappppppp) [inline EPlast/EPlast.BLL/Commands/Club/UnArchiveCommand.cs:11]: Same here

File: EPlast/EPlast.BLL/Commands/Club/UnArchiveCommand.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        public UnArchiveCommand(int cLubId)
```
Comment (ostappppppp) [inline EPlast/EPlast.BLL/Commands/Club/UnArchiveCommand.cs:12]: And here

File: EPlast/EPlast.BLL/MediatrEntryPoint.cs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace EPlast.BLL
+{
+    public class MediatrEntryPoint
```
Comment (ostappppppp) [inline EPlast/EPlast.BLL/MediatrEntryPoint.cs:7]: For what reason you need that empty class?

Comment (duhjent) [reply EPlast/EPlast.BLL/MediatrEntryPoint.cs:7]: For marking assembly. Maybe to make it more self-descriptive, can use some marker interface like `IBLLAssembly` (which can also be used with other assembly scanning libraries).

**Pull request closed without merging**